crates/uv/src/commands/build_frontend.rs

```diff
@@ -1140,13 +1148,17 @@ enum BuildMessage {
     Build {
         /// The name of the built distribution.
         filename: DistFilename,
```

Comment:
Do we still need to `filename` field now that we have the raw name?

Reply:
There are a couple places that grab the `.version()` and there's this line:
```
            let path = output_dir.join(sdist_build.filename().to_string());
```
I wasn't sure if it would be safe to replace it for that path.

Reply:
I would expect that `let path = output_dir.join(sdist_build.filename().to_string` needs to use the raw filename since we're trying to read from that path. Can we check why that _isn't_ failing today?

Reply:
I added a test to `build.rs` that fails because of this line. The test passes after updating to use the raw filename.